eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -10,7 +10,7 @@
   <Target Name="CreateTestWasmAppBundle"
           AfterTargets="Publish"
           DependsOnTargets="BundleTestWasmApp"
-          Condition="'$(TargetArchitecture)' == 'wasm' And '$(TargetOS)' == 'browser'" />
+          Condition="'$(TargetArchitecture)' == 'wasm' And ('$(TargetOS)' == 'browser' or '$(TargetOS)' == 'wasi')" />
```

Comment:
Isn't `TargetArchitecture is wasm` the same as `TargetOS is browser or wasi`? So each place that has this or `#if TARGET_BROWSER || TARGET_WASI` could use `TARGET_WASM`? Not saying it's better, just shorter.

Reply:
I would rather drop the TargetArchitecture part here